src/parser/token.ml

```diff
@@ -74,7 +74,8 @@ type t =
   | `Begin_style of style
   | `Begin_paragraph_style of paragraph_style
   | (* Other inline element markup. *)
-    `Simple_reference of string
+    `Simple_reference of
+    string
```

Comment:
This line break seems unnecessary!

Reply:
Fixed: https://github.com/ocaml-ppx/ocamlformat/pull/2606